tests/software/rasusa/main.nf

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+params.depth_cutoff = 100
+params.genome_size = '1000000b'
+
+include { RASUSA } from '../../../software/rasusa/main.nf' addParams( options: [:], depth_cutoff: params.depth_cutoff, genome_size:  params.genome_size)
+
+workflow test_rasusa {
+    def input = []
+    input = [ [ id:'test', single_end:false], // meta map
+              [file('https://raw.githubusercontent.com/nf-core/test-datasets/sarek/testdata/umi-dna/qiaseq/SRR7545951-small_1.fastq.gz', checkIfExists: true),
```

Comment:
I didn't even know this data existed! Does the SARS-CoV-2 data not work?

Reply:
just accidentally picked it

Reply:
Aha, but we still need to change this to the standard test data we are using like this e.g. 
https://github.com/nf-core/modules/blob/6927f1d086adda35fcc4899f3a27098ca268cef0/tests/software/fastqc/main.nf#L22-L25